x-pack/test/functional/es_archives/dashboard/legacy/mappings.json

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
This data I copied from `src/` to make `https://github.com/elastic/kibana/pull/60244/files#diff-401179577e069eaeb5f20de5ad1ba769R23` `await PageObjects.dashboard.initTests();` work. 
I didn't find how to load that data from `xpack`.